src/main/java/com/google/alpollo/AnalysisHelper.java

```diff
@@ -0,0 +1,34 @@
+package com.google.alpollo;
+
+import java.io.IOException;
+
+import com.google.api.gax.rpc.FixedHeaderProvider;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.LanguageServiceSettings;
+import com.google.cloud.language.v1.Sentiment;
+
+/**
+ * Helper class for cleaner code and easier testing.
+ */
+
+public class AnalysisHelper {
+  
+  /**
+   * Based on the lyrics sent, the AI can extract the main "sentiment" of the text.
+   * This sentiment has a score, showing the overall positivity of the text, ranging from -1 to 1
+   * and a magnitude, representing how strong the sentiment is, ranging from 0 to 1
```

Comment:
Magnitude changes from `0.0` to `+inf`.

From the documentation: 
`magnitude indicates the overall strength of emotion (both positive and negative) within the given text, between 0.0 and +inf`